.github/actions/install-dashboards/action.yml

```diff
@@ -33,28 +34,41 @@ runs:
         ref: 'main'
         fetch-depth: 0
 
-    - run: mkdir -p plugins
-      working-directory: OpenSearch-Dashboards
-      shell: bash
-
     - uses: actions/checkout@v2
       with:
         path: ${{ steps.determine-plugin-directory.outputs.plugin-directory }}
 
-    - id: osd-version
+    - id: osd-version-linux
+      if: ${{ runner.os == 'Linux' }}
       continue-on-error: true
```

Comment:
I don't think this should continue on error, lets remove this

Reply:
Fixed.